src/xrGame/Weapon.cpp

```diff
@@ -35,6 +35,8 @@
 constexpr pcstr WPN_SCOPE = "wpn_scope";
 constexpr pcstr WPN_SILENCER = "wpn_silencer";
 constexpr pcstr WPN_GRENADE_LAUNCHER = "wpn_launcher";
+// SoC Support - APAMk2
+constexpr pcstr WPN_GRENADE_LAUNCHER_SOC = "wpn_grenade_launcher";
```

Comment:
You will be in the project readme, contributors list and commit history, mentioning your changes using comments in the code is outdated practice :(

So this is better:
```suggestion
constexpr pcstr WPN_GRENADE_LAUNCHER_SOC = "wpn_grenade_launcher";
```